// echarts-modules/index.js
exports.chart = [
  'bar',
  'boxplot',
  'candlestick',
  'chord',
  'custom',
  'effectScatter',
  'funnel',
  'gauge',
  'graph',
  'heatmap',
  'line',
  'lines',
  'map',
  'parallel',
  'pictorialBar',
  'pie',
  'radar',
  'sankey',
  'scatter',
  'themeRiver',
  'tree',
  'treemap'
]
exports.component = [
  'angleAxis',
  'axis',
  'axisPointer',
  'brush',
  'calendar',
  'dataZoom',
  'dataZoomInside',
  'dataZoomSelect',
  'geo',
  'graphic',
  'grid',
  'gridSimple',
  'legend',
  'legendScroll',
  'markArea',
  'markLine',
  'markPoint',
  'parallel',
  'parallelAxis',
  'polar',
  'radar',
  'radiusAxis',
  'singleAxis',
  'timeline',
  'title',
  'toolbox',
  'tooltip',
  'visualMap',
  'visualMapContinuous',
  'visualMapPiecewise'
]
exports.types = {
  bar: 'bar',
  boxplot: 'boxplot',
  candlestick: 'candlestick',
  chord: 'chord',
  custom: 'custom',
  effectScatter: 'effectScatter',
  funnel: 'funnel',
  gauge: 'gauge',
  graph: 'graph',
  heatmap: 'heatmap',
  line: 'line',
  lines: 'lines',
  map: 'map',
  parallel: 'parallel',
  pictorialBar: 'pictorialBar',
  pie: 'pie',
  radar: 'radar',
  sankey: 'sankey',
  scatter: 'scatter',
  themeRiver: 'themeRiver',
  tree: 'tree',
  treemap: 'treemap',
  angleAxis: 'angleAxis',
  axis: 'axis',
  axisPointer: 'axisPointer',
  brush: 'brush',
  calendar: 'calendar',
  dataZoom: 'dataZoom',
  dataZoomInside: 'dataZoomInside',
  dataZoomSelect: 'dataZoomSelect',
  geo: 'geo',
  graphic: 'graphic',
  grid: 'grid',
  gridSimple: 'gridSimple',
  legend: 'legend',
  legendScroll: 'legendScroll',
  markArea: 'markArea',
  markLine: 'markLine',
  markPoint: 'markPoint',
  parallelAxis: 'parallelAxis',
  polar: 'polar',
  radiusAxis: 'radiusAxis',
  singleAxis: 'singleAxis',
  timeline: 'timeline',
  title: 'title',
  toolbox: 'toolbox',
  tooltip: 'tooltip',
  visualMap: 'visualMap',
  visualMapContinuous: 'visualMapContinuous',
  visualMapPiecewise: 'visualMapPiecewise'
}
